### ISSUE TITLE
[help.keyman.com] Font License, Font documentation links are not working in Ubuntu 22.04 and Lunar 23.04 Linux OS

### ISSUE THREAD

**Issue by bharanidharanj** [keymanapp/help.keyman.com#811]
Here, what I did is:
1. Installed Syriac/Aramaic keyboard using Keyman Configuration dialog. 
2. Clicked 'Syriac/Aramaic' Keyboard from the Keyboard Layouts view. 
3. Clicked 'Help' button. 
4. In the syriac_aramaic keyboard package help page, Click the Aramaic (Hebrew Layout) blue link button. 
5. Click the 'Font license' blue link button. 
I noticed that it is opening an 'Access denied' page. 
6. Click the 'Font documentation' blue link button. 
It is showing the same 'Access denied' page. 

I have attached the Screenshot for reference. 

   <img width="300" src="https://github.com/keymanapp/help.keyman.com/assets/19683143/b0fc1df0-5eaf-409e-b33a-9cb9906f3962">

Note: In Ubuntu 20.04, Wasta 20.04 Linux, it is working fine. ie., the help page opens without showing any error message.

**Comment by darcywong00** [comment]
Maybe @ermshiperete can confirm if Ubuntu 22.04 and onward uses the snap package of Firefox.

I searched and found this
https://bugs.launchpad.net/snapd/+bug/1972762/comments/2

> Thank you for reporting this but, this is by design. Snaps can't access the user's home by default.

The launchpad comment goes on to include some possible workarounds

**Comment by ermshiperete** [comment]
Yes, the reason is that by default Ubuntu 22.04+ installs Firefox as snap package which can't access files under `$HOME/.local` (non-hidden directories/files, i.e. which don't start with `.`, would work though).

You'll get the same error if you click on the "Open in Web browser" button when displaying the Help for a keyboard.

Unfortunately the mentioned workarounds don't really help us because they require changes to the snap package itself.
